nyxx.interactions/lib/src/models/CommandInteractionOptionType.dart

```diff
@@ -0,0 +1,12 @@
+part of nyxx_interactions;
+
+enum CommandInteractionOptionType {
```

Comment:
I don't like enums in Dart because they are pretty limiting. I think it will be fine in that case, but it can cause a problem if we will need to attach value to it.

Reply:
Removed enums using raw int value from API / Gateway as of next push